### PR TITLE
Enable pylint `no-member` check

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
Easy to finish up this stale branch I had enabling `no-member` as a pylint check after all the work @mscuthbert did typing v8!

Some low impact bugs fixed. Not too many disables. If it's too many we can just pull in the fixes. The commits are individual if that helps to review.

### CODE REVIEW

File: music21/scale/intervalNetwork.py
Comment (mscuthbert) [inline music21/scale/intervalNetwork.py:2330]: :-O

File: music21/test/timeGraphs.py
Comment (mscuthbert) [inline music21/test/timeGraphs.py:31]: these lines can be eliminated now, I believe.

**Review by mscuthbert — APPROVED** [review]
APPROVED but don't merge yet -- I think that this will bite some of Greg's code, and I'd rather not throw one more wrench into that.  After DC is merged, we can visit this again.  THANKS!

**Comment by mscuthbert** [comment]
Thanks!  Really helpful!

**Review by hikmaharomire — COMMENTED** [review]
Enable pylint no-member check

**Pull request merged**
